tests/aslts/bin/Do

```diff
@@ -341,7 +348,12 @@ do_test_cases_make_install()
 
 		if [ $3 != 0 ]; then
 			cd "$dir"
-			make install > /dev/null
+			if [ "$execonly" = "yes" ]; then
```

Comment:
A useful trick in string comparison in shell is:

if [ "x$execonly" = "xyes" ]

The extra "x" ensures no exceptions when environment of execonly doesn't exist.